Source/Scene/GroundPrimitive.js

```diff
@@ -807,6 +903,19 @@ define([
         }
     }
 
+    GroundPrimitive._initialized = false;
+    GroundPrimitive.init = function() {
```

Comment:
This function is just hacked in there. Once we figure out how we are actually going to set this up I can rename and optimize the function.